etp-backend/src/main/clj/solita/etp/api/valvonta_oikeellisuus.clj

```diff
@@ -76,13 +80,15 @@
                      :responses  {200 {:body nil}}
                      :handler    (fn [{{{:keys [id]} :path :keys [body]}
                                        :parameters :keys [db whoami]}]
-                                   (r/response (valvonta-service/save-valvonta! db id body)))}}]
+                                   (api-response/ok|not-found
+                                     (valvonta-service/save-valvonta! db id body)
+                                     (str "Energiatodistus " id " does not exists.")))}}]
 
      ["/toimenpiteet"
       [""
        {:get  {:summary    "Hae energiatodistuksen valvontatoimenpiteet."
                :parameters {:path {:id common-schema/Key}}
-               :responses  {200 {:body [oikeellisuus-schema/Toimenpide]}}
+               :responses  {200 {:body [(dissoc oikeellisuus-schema/Toimenpide :virheet)]}}
```

Comment:
Tämäkin voisi olla siistimpi schema-nimiavaruudessa, mutta makuasia.

Reply:
Olen jättänyt pois joiltakin palveluilta toimenpiteen tietoa, jota tällä hetkellä ei tarvita kälissä. Näitä ei ole mukava tuoda schema ns:ään koska sitten näille pitäisi keksiä nimi ja ko. ns. tulee sekava. Lisäksi näille erikoistapauksille on tyypillisesti vain yksi käyttökohta. Jos käyttökohtia on enemmän kuin yksi niin sitten laittaisin skeemaan muuten en.

Reply:
Tässä kannattaa miettiä myös luettavuutta eli kumpi on selkeämpi skeema:
(dissoc oikeellisuus-schema/Toimenpide :virheet) vai oikeellisuus-schema/ToimenpideLightVersion1